prayer_times_calculator/pray_times_calculator.py

```diff
@@ -72,10 +86,29 @@ def __init__(
         self._lat_adj_method = self.LAT_ADJ_METHODS.get(
             latitudeAdjustmentMethod.lower()
         )
+        if tune is True:
+            self._tune = str(imsak_tune) + ',' + str(fajr_tune) + ',' \
+                + str(sunrise_tune) + ',' + str(dhuhr_tune) + ',' \
+                + str(asr_tune) + ',' + str(maghrib_tune) + ',' \
+                + str(sunset_tune) + ',' + str(isha_tune) + ',' \
+                + str(midnight_tune)
+        else:
+            self._tune = False
+
+        if self._calculation_method == 99: 
+            self.custom_method(fajr_angle, maghrib_angle, isha_angle)
+        else: self._method_settings = False
 
         date_parsed = datetime.strptime(date, "%Y-%m-%d")
         self._timestamp = int(date_parsed.timestamp())
 
+    def custom_method(self, fajr_angle, maghrib_angle, isha_angle):
+            if fajr_angle is None: fajr_angle = "null"
+            if maghrib_angle is None: maghrib_angle = "null"
+            if isha_angle is None: isha_angle = "null"
+            self._method_settings = str(fajr_angle) + ',' + str(maghrib_angle) \
```

Comment:
you're setting all the angles to string already so I don't think we need to convert to string here, correct?

Reply:
nope. they raised error when i try to ommit the conversion. 
```python 
TypeError: unsupported operand type(s) for +: 'int' and 'str' 
```